app/views/users/lists/show.html.erb

```diff
@@ -31,6 +31,13 @@
                          :autocomplete => "on",
                          :class => "form-control" %>
     </div>
+    <div class="mb-3 col-md-auto">
+      <%= select_tag :edits,
+                     options_for_select([["Has Edits", "yes"], ["No Edits", "no"]], params[:edits]),
```

Comment:
There's no `t("...")` for "Has Edits", "No Edits", although there is `t(".edits")` on the next line.

Reply:
Fixed.